internal/utils/utils.go

```diff
@@ -1,9 +1,13 @@
 package utils
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type KeyMapType = map[string]interface{}
```

Comment:
WDYT about `Set` ?

Reply:
What exactly do you mean here?

Reply:
I mean `StringSet` would be more descriptive name of the data structure than `KeyMapType`